Makefile

```diff
@@ -55,6 +55,8 @@ make platform      # build for specific Linux distribution
   ARTIFACTS=1        # copy artifacts from docker image
   PUBLISH=1          # publish (i.e. docker push) after build
 
+make sanbox        # create container with CLang Sanitizer
```

Comment:
Need to mention the /w arg for volume?

Reply:
It's a sort of hidden feature at the moment. :-)